src/mpyl/steps/deploy/k8s/__init__.py

```diff
@@ -122,16 +122,20 @@ def substitute_namespaces(
     """
     env = env_vars.copy()
 
-    def get_namespace_for_linked_project(project_name: ProjectName):
+    def get_namespace_for_linked_project(project_name: ProjectName) -> Optional[str]:
         is_part_of_same_deploy_set = project_name in projects_to_deploy
         if is_part_of_same_deploy_set and pr_identifier:
             return f"pr-{pr_identifier}"
         return project_name.namespace
```

Comment:
I think we can make the namespace non-optional if we adjust the logic here:
[get_namespace_for_linked_project](https://github.com/Vandebron/mpyl/blob/876c21a5c4b8d7f04e1322d83b397c865325679b/src/mpyl/project.py#L409-L410)
using the same logic as before, with the fallback on `project.name` @Jorg88

Reply:
done in [90e307f](https://github.com/Vandebron/mpyl/pull/200/commits/90e307fc80c5e733e36f7ba7dafc39ae5c5e9622)